site/scripts/library.js

```diff
@@ -404,6 +404,50 @@ function showMangaDetails(manga, categories, source) {
     }
   })();
 
+  ///////////////////
+  const trackingImages = [
+    { src: 'img/trackers/ic_tracker_mal.webp', greyed: true },
+    { src: 'img/trackers/ic_tracker_anilist.webp', greyed: true },
+    { src: 'img/trackers/ic_tracker_kitsu.webp', greyed: true },
+    { src: 'img/trackers/ic_manga_updates.webp', greyed: true },
+    { src: 'img/trackers/ic_tracker_shikimori.webp', greyed: true },
+    { src: 'img/trackers/ic_tracker_bangumi.webp', greyed: true },
+  ];
+
+  // Extract sync IDs if manga.tracking exists
+  const syncIds = manga.tracking
+    ?.map((track, index) => ({ syncId: track?.syncId, trackingUrl: track?.trackingUrl, index }))
+    .filter(item => item.syncId != null);
+
+  // Update trackingImages based on syncIds
+  syncIds?.forEach(item => {
+    if (item.syncId >= 1 && item.syncId <= 6) {
+      trackingImages[item.syncId - 1].greyed = false;
+      trackingImages[item.syncId - 1].trackingUrl = item.trackingUrl;
+      trackingImages[item.syncId - 1].index = item.index;
+    }
+  });
```

Comment:
For reference, [the IDs are](https://github.com/jobobby04/TachiyomiSY/blob/f6fc2d7e2f646ea48978d1b80749e3fc03147668/app/src/main/java/eu/kanade/tachiyomi/data/track/TrackerManager.kt):
```kotlin
    val myAnimeList = MyAnimeList(1L)
    val aniList = Anilist(2L)
    val kitsu = Kitsu(3L)
    val shikimori = Shikimori(4L)
    val bangumi = Bangumi(5L)
    val komga = Komga(6L)
    val mangaUpdates = MangaUpdates(7L)
    val kavita = Kavita(8L)
    val suwayomi = Suwayomi(9L)
    val mdList = MdList(60L)  // SY
```

So might wanna change this index-based approach